k8s/namespaces/jenkins/patches/cftptl/cluster-00/jenkins.yaml

```diff
@@ -19,8 +19,8 @@ spec:
           cpu: "4"
           memory: "22Gi"
         limits:
-          cpu: "6"
-          memory: "22Gi"
+          cpu: "10"
+          memory: "25Gi"
```

Comment:
Not sure if this will actually work, why are you doing this change?

Reply:
I don't think it's going to work. I've tried already

Reply:
This was to fix the Jenkins builds waiting for the executors and noticing the resource crunch. Yeah as discussed on the Slack I would close this one.